app/views/handlers/individual_response.py

```diff
@@ -844,7 +845,7 @@ def handle_get(self):
     def handle_post(self):
         if self.selected_option == self.confirm_option:
             self._update_section_status(CompletionStatus.INDIVIDUAL_RESPONSE_REQUESTED)
-            self._publish_fulfilment_request(self.mobile_number)
+            self._publish_fulfilment_request(sanitise_mobile_number(self.mobile_number))
```

Comment:
Had a chat with @ajmaddaford and come to a conclusion that this should probably be done when the fulfilment request payload is created. Otherwise, we have no guard against the IR payload containing an unsanatized number given we can't integration test the IR journey while checking the payload.